src/Loading/babylon.sceneLoader.ts

```diff
@@ -1,16 +1,27 @@
 ﻿module BABYLON {
+    export interface ISceneLoaderPluginExtensions {
+        [extension: string]: {

```

Comment:
Yes, each extension can now specify an isBinary flag. If false, it will load the file into a string as it did before. If true, it will load the file into an array buffer for easier binary parsing.